stable/stolon/values.yaml

```diff
@@ -77,6 +91,10 @@ keeper:
   tolerations: []
   volumes: []
   volumeMounts: []
+  podDisruptionBudget:
+    enabled: true
+    # minAvailable: 1
+    maxUnavailable: 1
```

Comment:
You should not set defaults for PDBs here. Otherwise a user would have to explicitly unset `maxUnavailable` if they wanted to set `minAvailable` because you cannot set both.

In case you like it, here's an alternative way of configuring this:
* https://github.com/helm/charts/blob/master/stable/keycloak/values.yaml#L145-L147
* https://github.com/helm/charts/blob/master/stable/keycloak/templates/poddisruptionbudget.yaml#L16

Reply:
Removed default pdb values and disabled by default.